cafebabel/templates/_social-networks-macro.html

```diff
@@ -1,11 +1,12 @@
 {% macro social_networks(with_linkedin=False) %}
   <ul>
-    <li><a href={{ social_network_url_for('facebook') }} class=icon-facebook></a></li>
-    <li><a href={{ social_network_url_for('twitter') }} class=icon-twitter></a></li>
-    <li><a href={{ social_network_url_for('instagram') }} class=icon-instagram></a></li>
-    <li><a href={{ social_network_url_for('youtube') }} class=icon-youtube></a></li>
+    <li><a href={{ social_network_url_for('facebook') }} class=icon-facebook title="Cafébabel on Facebook"></a></li>
+    <li><a href={{ social_network_url_for('twitter') }} class=icon-twitter title="Cafébabel on Twitter"></a></li>
+    <li><a href={{ social_network_url_for('instagram') }} class=icon-instagram title="Cafébabel on Instagram"></a></li>
+    <li><a href=#static-pages-links class=icon-newsletter title="Cafébabel on Facebook"></a></li>
```

Comment:
I think that title is wrong 😉

Reply:
rooo https://github.com/cafebabel/cafebabel.com/pull/386/commits/a26d84cb31680b7f0dfb19db76d0580a82af6628